quetz/utils.py

```diff
@@ -39,6 +39,47 @@ def check_package_membership(package_name, includelist, excludelist):
     return True
 
 
+def _include_pattern_match(name, version, build, pattern):
```

Comment:
This looks a lot like a MatchSpec - I think we should just use that :)

That means we should either depend on conda or vendor the MatchSpec part from conda into quetz. I think there were also discussions on making MatchSpec standalone but not sure if they ever got anywhere.

Reply:
Nice catch!
We actually have `conda` as a dependency already because of `conda-build` and `mamba` apparently.

I'm now using MatchSpec and since it also works for _just_ the package name, we might be able to get rid of the additional fields `include_pattern_list` and just use the `include_list` again, making the transition even simpler.